package.json

```diff
@@ -8,16 +8,13 @@
   },
   "homepage": "https://react-bootstrap.github.io/",
   "scripts": {
-    "build": "babel-node tools/build-cli.js",
-    "docs": "babel-node docs/dev-run.js",
-    "docs-build": "babel-node tools/build-cli.js --docs-only",
-    "docs-prod": "npm run docs-build && cross-env NODE_ENV=production babel-node docs/server.js",
-    "docs-prod-unoptimized": "npm run docs-build -- --dev && cross-env NODE_ENV=production babel-node docs/server.js",
-    "lint": "eslint src test tools webpack *.js",
+    "build": "node tools/build.js",
+    "build-docs": "npm run build lib && cd www && npm i && npm run build",
+    "lint": "eslint www/src src test tools webpack *.js",
     "release": "release",
     "tdd": "karma start",
     "test": "npm run lint && npm run test-browser && npm run test-node",
-    "test-browser": "cross-env NODE_ENV=test karma start --single-run",
```

Comment:
iirc there was something glitchy with the dev-expression transform when we didn't do this